File: Sprint/Sprint/Models/ModelBacklogItem/BacklogItemDTO.cs
using Sprint.Models.ModelBacklog;
using Sprint.Models.ModelPOBI;
using Sprint.Models.ModelSprint;
using System.ComponentModel.DataAnnotations.Schema;

namespace Sprint.Models.ModelBacklogItem
{
    public class BacklogItemDTO
    {
        public Guid BacklogItemId { get; set; }
        public string TimeAddedBacklogItem { get; set; }


        public Guid BacklogId { get; set; }
        public BacklogB Backlog { get; set; }

        public Guid SprintId { get; set; }
        public SprintS Sprint { get; set; }

        public Guid POBIId { get; set; }
        public PhaseOfBacklogItem POBI { get; set; }
    }
}

File: FrontendTemplateProject-main/src/app/models/backlogItemDTO.ts
import { Guid } from 'guid-typescript';

export class BacklogItemDTO {
    
    backlogItemId!: Guid;
    timeAddedBacklogItem: string;
    backlogId!: Guid;
    sprintId!: Guid;
    pobiId!: Guid;
    
    constructor(
        backlogItemId: Guid,
        timeAddedBacklogItem: string,
        backlogId: Guid,
        sprintId: Guid,
        pobiId: Guid
    ) {
        this.backlogItemId = backlogItemId;
        this.timeAddedBacklogItem = timeAddedBacklogItem;
        this.backlogId = backlogId;
        this.sprintId = sprintId;
        this.pobiId = pobiId;
    }
    
}